pepper-apis/src/main/java/org/broadinstitute/ddp/service/PdfGenerationService.java

```diff
@@ -138,16 +139,28 @@ public InputStream generateFlattenedPdfForConfiguration(PdfConfiguration configu
                                        Handle handle) throws IOException {
         Participant participant = loadParticipantData(handle, configuration, userGuid);
         Map<Long, ActivityResponse> instances = loadActivityInstanceData(handle, configuration, participant);
+        Long languageId = null;
+        if (participant.getUser().getProfile() != null) {
+            languageId = participant.getUser().getProfile().getPreferredLangId();
+        }
+        if (languageId == null) {
+            //use default 'en'
+            languageId = LanguageStore.getDefault().getId();
```

Comment:
Could we perhaps figure the study's "default" language and use that? Might avoid situations where our platform default is English but the study doesn't have English and their default is instead French or something.